core/modules/views/includes/view.inc

```diff
@@ -5,15 +5,15 @@
  */
 
 /**
- * @defgroup views_objects Objects that represent a View or part of a view
+ * @defgroup views_objects Objects that represent a View, or part of a view
  * @{
- * These objects are the core of Views do the bulk of the direction and
+ * These objects are the core of Views, do the bulk of the direction and
  * storing of data. All database activity is in these objects.
```

Comment:
This doesn't make sense to me... I think it should either be "These objects are the core of Views; they do the bulk of the direction and storing of data." or "These objects are the core of Views and do the bulk of the direction and storing of data."

Reply:
Agreed 👍 ...I went with the semicolon version.